ksql-udf-quickstart/src/main/resources/archetype-resources/src/main/java/ReverseUdf.java

```diff
@@ -17,27 +17,45 @@
 
 import io.confluent.ksql.function.udf.Udf;
 import io.confluent.ksql.function.udf.UdfDescription;
+import io.confluent.ksql.function.udf.UdfParameter;
 
-@UdfDescription(name = "reverse", description = "Example UDF that reverses an object")
+@UdfDescription(
+    name = "reverse",
+    description = "Example UDF that reverses an object",
+    version = "${version}",
+    author = "${author}"
+)
 public class ReverseUdf {
 
   @Udf(description = "Reverse a string")
-  public String reverseString(String source) {
+  public String reverseString(
+      @UdfParameter(value = "source", description = "the value to reverse")
```

Comment:
If the value is omitted, then the parameter name will use the same variable name. 
Should we take advantage of that? It is not on the UDF doc either.

Reply:
Yeah, that's true, but only the case if the code is compiled with `-parameters`, which may not be the case.  Better to have the example provide the name IMHO.